app/code/Magento/Checkout/view/frontend/web/js/view/summary/shipping.js

```diff
@@ -22,13 +22,18 @@ define([
          */
         getShippingMethodTitle: function () {
             var shippingMethod;
+            var shippingMethodTitle = '';
 
             if (!this.isCalculated()) {
                 return '';
             }
             shippingMethod = quote.shippingMethod();
 
-            return shippingMethod ? shippingMethod['carrier_title'] + ' - ' + shippingMethod['method_title'] : '';
+            if (typeof(shippingMethod['method_title']) !== 'undefined') {
+                shippingMethodTitle = ' - ' + shippingMethod['method_title'];
+            }
+
+            return shippingMethod ? shippingMethod['carrier_title'] + shippingMethodTitle : '';
```

Comment:
If I understand correctly, your code will return empty string if no `method_title` is found. Can we make it return `carrier_title` only in that case?